metrics/system.go

```diff
@@ -0,0 +1,53 @@
+package metrics
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	metrics "github.com/rcrowley/go-metrics"
+	"github.com/shirou/gopsutil/process"
+)
+
+// CPUTime returns an amount of CPU time (in seconds) assigned to executor process
+// by system.
+func CPUTime() (float64, error) {
+	p := getExecutorProcess()
+	t, err := p.Times()
+	if err != nil {
+		return 0, fmt.Errorf("Unable to get CPU times: %s", err)
+	}
+
+	total := t.User + t.System + t.Nice + t.Iowait + t.Irq + t.Softirq +
```

Comment:
Use [`t.Total`](https://godoc.org/github.com/shirou/gopsutil/cpu#TimesStat.Total) instead

Reply:
Or add comment why not using it

Reply:
Commented.